common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -192,9 +192,12 @@ public static boolean tryLoadItemList() {
         handler.addRequest(new RequestBuilder(apiUrls.get("Athena") + "/cache/get/itemList", "item_list")
                 .cacheTo(new File(API_CACHE_ROOT, "item_list.json"))
                 .handleJsonObject(json -> {
-                    translatedReferences = gson.fromJson(json.getAsJsonObject("translatedReferences"), HashMap.class);
-                    internalIdentifications =
-                            gson.fromJson(json.getAsJsonObject("internalIdentifications"), HashMap.class);
+                    @SuppressWarnings("unchecked")
```

Comment:
If you think this is worth handling (and I'm tempted to agree, unchecked warnings are bad), using TypeTokens is probably better. See e.g. https://github.com/google/gson/blob/master/UserGuide.md#collections-examples